src/pages/[...slug].tsx

```diff
@@ -83,13 +87,14 @@ const DocPage: NextPage<Props> = ({ frontmatter, content }) => {
       />
 
       <main>
-        <Stack py={8} px={4}>
+        <Stack mb={16}>
           <Breadcrumbs router={router} />
-
-          <Heading as='h1'>{frontmatter.title}</Heading>
-
-          <ReactMarkdown components={MDXComponents}>{content}</ReactMarkdown>
+          <Heading as='h1' mt='4 !important' mb={0} {...textStyles.header1}>
+            {frontmatter.title}
+          </Heading>
+          {/* <Text as='span' mt='0 !important'>last edited {TODO: get last edited date}</Text> */}
```

Comment:
is this done?

Reply:
AFAIK we don't have a mechanism to pull the last edited date yet. Did not attempt that for this PR.

Reply:
I will add this, can get in getStaticProps method.

Reply:
https://github.com/ethereum/geth-website/issues/58 created an issue for this

Reply:
Awesome, thanks @corwintines!